vesicles/validations/vesreac_error.py

```diff
@@ -126,7 +126,7 @@ def run_sim(hdfPath, ntets, D, NITER, use_foi, use_soAA, seed):
         import subprocess
 
         nmpi, totproc, nrunsPerProc, baseseed = map(int, sys.argv[2:])
-        hdfPrefix = 'data/vesreac_error'
+        hdfPrefix = 'data/vesreac_error/vesreac_error'
```

Comment:
The files do not appear to have been moved on this PR

Reply:
I've included the .zip file and when you unzip, by default it extracts to` data/vesreac_error/`

Reply:
I see, so the `data/vesreac_error_xxx.h5` files can be removed then?

Reply:
We don't have any of those files.

Reply:
Ah right, I just have them locally. We do have `data/vesreac_immobile_reactants_xxx.h5` that I added at some point, I will zip them and modify the scripts like you did for `vesreac_error`.

Reply:
We can, but actually they are not as much data as the vesreac_error one so I left them.